tooling/reproducible/windows_repro_build_compare.sh

```diff
@@ -39,8 +39,8 @@ TARBALL_URL="$2"
 # These Values Should Be Updated To Reflect The Build Environment
 # The Defaults Below Are Suitable For An Adoptium Windows Build Environment
 # Which Has Been Created Via The Ansible Infrastructure Playbooks
-CURR_DIR=$(pwd)
-WORK_DIR="$CURR_DIR/cmp$(date +%Y%m%d%H%M%S)"
+
+WORK_DIR="C:/workspace/comp-jdk-build/"
```

Comment:
Would this dir be available for dev's local environment?

Reply:
Yes, the script creates it, if it doesn't exist...

Reply:
Ah, but I can probably make this better...

Reply:
Let me make a quick fix..

Reply:
should be ok now.